module/server.cpp

```diff
@@ -19,6 +19,8 @@ PluggablePhysicsServer::PluggablePhysicsServer() {
 
 PluggablePhysicsServer::~PluggablePhysicsServer() {
 	this->library->terminate();
+  memdelete(this->body_state_singleton);
+  memdelete(this->space_state_singletone);
```

Comment:
This project uses tabs for indentation, not spaces.

Reply:
Also, there is a stray 'e' attached to the space_state_singletone

Reply:
Sorry about that, fixed.

Reply:
fixed, I guess I need to get some sleep.